### PR TITLE
Adding a Dockerfile and instructions on how to use it

### PR DESCRIPTION
**Pull request by isleshocky77**
This adds a Dockerfile and updates the README on how to use import-mail-to-gmail under Docker. This allows for the ability to not have to worry about setting up or having conflicts of Python on the local system.

As noted in the README.

## Build

```
docker build -t google/import-mailbox-to-gmail .
```

### Import directory

```
docker run --rm -it \
  -v "/local/path/to/auth.json:/auth.json" \
  -v "/local/path/to/mbox:/mbox" google/import-mailbox-to-gmail \
  --json "/auth.json" --dir "/mbox"
```

### Display the Help Banner

```
docker run --rm -it google/import-mailbox-to-gmail
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:2
+
+WORKDIR /usr/src/app
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY import-mailbox-to-gmail.py .
+
+ENTRYPOINT [ "python", "./import-mailbox-to-gmail.py" ]
```
Comment (eesheesh) [inline Dockerfile]: Is the `./` required?

File: README.md [hunk 2]
```diff
@@ -159,3 +159,4 @@ viewing the `import-mailbox-to-gmail.log` file.
 * To import mail from Apple Mail.app, make sure you export it first - the raw Apple Mail files can't be imported. You can export a folder by right clicking it in Apple Mail and choosing "Export Mailbox".
 * This script can import nested folders. In order to do so, it is necessary to preserve the email folders' hierarchy when exporting them as mbox files. In Apple Mail.app, this can be done by expanding all subfolders, selecting both parents and subfolders at the same time, and exporting them by right clicking the selection and choosing "Export Mailbox". 
 * If any of the folders have a ".mbox" extension and a file named "mbox" in them, the contents of the "mbox" file will be imported to the label named as the folder. This is how Apple Mail exports are structured.
+* To run under [Docker](https://www.docker.com/): First build the image, `docker build -t google/import-mailbox-to-gmail .`; then run the import command, `docker run --rm -it -v "/local/path/to/auth.json:/auth.json" -v "/local/path/to/mbox:/mbox" google/import-mailbox-to-gmail --json "/auth.json" --dir "/mbox"`
```
Comment (eesheesh) [inline README.md]: 1. I suggest breaking it down to make it more readable. Note that the syntax in my suggestion might be a bit off. :) I suggest adding numbered steps and putting each example command in its own line.

```suggestion
* To run under [Docker](https://www.docker.com/):
   1. Build the image:
       `docker build -t google/import-mailbox-to-gmail .`
   2. Run the import command:
     `docker run --rm -it -v "/local/path/to/auth.json:/auth.json" -v "/local/path/to/mbox:/mbox" google/import-mailbox-to-gmail --json "/auth.json" --dir "/mbox"`
```

2. Please explain the `/path/to/file.json:file.json` syntax in the README, as it it's unusual compared to running it normally from the terminal.

File: Dockerfile (hunk 1 above)
Comment (isleshocky77) [reply Dockerfile]: Removed.

File: README.md (hunk 2 above)
Comment (isleshocky77) [reply README.md]: I completely agree. I just didn't know if this should be more of an after thought / side note, or take up more space in the README. I've expanded it out a bit.

**Comment by eesheesh** [comment]
Thanks!

**Pull request merged**